src/menu/menu.scss

```diff
@@ -171,6 +173,7 @@ $iui-active-outline: thin solid t(iui-color-foreground-primary);
 @mixin iui-menu-icon {
   width: $iui-icons-default;
   height: $iui-icons-default;
+  display: flex;
```

Comment:
Could not find, where it has influence..

Reply:
Removed